lib/node_modules/@stdlib/math/base/special/besselj0/README.md

```diff
@@ -12,6 +12,8 @@ The [Bessel function of the first kind][bessel-first-kind] of order zero is defi
 
 </section>
 
+<!-- /.intro -->
```

Comment:
Whoopsie. Well this one needs it too.

Reply:
Yeah, went ahead and added it in the PR when I had to resolve the merge conflicts. :)